### ISSUE TITLE
Check the front-end can handle a collection path that doesn't have a label

### ISSUE THREAD

**Issue by alexwlchan** [wellcomecollection/platform#5308]

**Comment by jamesgorrie** [comment]
[These are the types](https://github.com/wellcomecollection/wellcomecollection.org/blob/b57e7ff3b2821cdc8fd7c45ceb3806f040edd654/common/model/catalogue.ts#L23-L24) we're using so it should be able to - but searching the code base we never use it but rather the `referenceNumber`.

**Comment by alexwlchan** [comment]
I'm not sure those fields in the front-end model are ever populated in a DisplayWork. 🤔 

The referenceNumber is where CollectionPath.label ends up so as long as the tree can cope without it we'll be fine, but I don't think this field has ever been empty in Calm so we might not have tested it.

**Comment by jamesgorrie** [comment]
We don't use it anywhere, and actually just read `referenceNumber`.

**Comment by alexwlchan** [comment]
Yeah, collectionPath doesn’t end up in the API at all, and I’ve removed it from the front-end codebase.
